stores/mssql/src/storage/domains/memory/index.ts

```
@@ -689,12 +690,50 @@ export class MemoryMSSQL extends MemoryStorage {
+              metadataParams[paramName] = value ? 'true' : 'false';
+            } else {
+              metadataParams[paramName] = String(value);
+            }
+          }
+          metadataIndex++;
+        }
+      }
+
+      const actualWhereClause = baseWhereClause + metadataWhereClause;
       const bindWhereParams = (req: sql.Request) => {
         Object.entries(whereParams).forEach(([paramName, paramValue]) => req.input(paramName, paramValue));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)